pkg/reconciler/v1alpha1/revision/reconcile_resources.go

```diff
@@ -58,9 +58,15 @@ func (c *Reconciler) reconcileDeployment(ctx context.Context, rev *v1alpha1.Revi
 	} else if getDepErr != nil {
 		logger.Errorf("Error reconciling deployment %q: %v", deploymentName, getDepErr)
 		return getDepErr
+	} else {
+		// The deployment exists, but make sure that it has the shape that we expect.
+		var err error
```

Comment:
I don't think the coded uses getDepErr and err at the same time, so you can avoid `var err error` here and above, by basically having deployment, err := ...

Reply:
This sets `deployment` :)

Reply:
Sorry, I wasn't clear.
Above do d, err := <>
elsewhere do err =, deployment, err =